agent/doc/swdesign/plantuml/seq_update_workload_complete.plantuml

```diff
@@ -0,0 +1,67 @@
+@startuml seq_update_workload_initial
```

Comment:
Is this diagram used somewhere? There is not SVG for this diagram and I do not see any reference to this diagram in the swdesign (of the Agent)

Reply:
Seems like it's some leftover from the initial design phase. I'll remove it.